src/Application/UI/Presenter.php

```diff
@@ -21,6 +21,7 @@
  * @property-read string $action
  * @property      string $view
  * @property      string $layout
+ * @property-read ITemplate|\stdClass $template
```

Comment:
But in control it is not \stdClass, so PhpStorm shows every declaration of template variable as warning.